CRM/Commitcivi/Logic/StripeMigration.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+class CRM_Commitcivi_Logic_StripeMigration {
+    
+
+    /**
+     * @param \CRM_Commitcivi_Model_Event $event
+     *
+     * @return integer
+     * @throws \CiviCRM_API3_Exception
+     */
+    public function migrate(CRM_Commitcivi_Model_Event $event)
+    {
+        $contact = $event->contact;
+        $donation = $event->donation;
+        
+        # find the current houdini recurring donation using email, processor, amount and start_date
```

Comment:
Wouldn't it be easier and more reliable to look up base on transaction id, which is supposed to be set with `$event->donation->recurringId`?
https://github.com/WeMoveEU/commitcivi/blob/master/CRM/Commitcivi/Logic/DonationStripe.php#L132

Reply:
Argh! Of course - I'll take a look.